packages/build/tests/telemetry/snapshots/tests.js.md

```diff
@@ -45,7 +44,77 @@ Generated by [AVA](https://ava.li).
       Netlify Build Complete                                        ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
-    (Netlify Build completed in 1ms)`
+    (Netlify Build completed in 1ms)␊
+    ␊
+    Error monitoring payload:␊
+    {␊
+      "errorClass": "telemetry",␊
+      "errorMessage": "Timeout awaiting 'request' for 1ms",␊
+      "context": "Telemetry error",␊
+      "groupingHash": "Telemetry error/nTimeout awaiting 'request' for 1ms",␊
+      "severity": "error",␊
+      "unhandled": true,␊
+      "pluginPackageJson": false,␊
+      "other": {␊
+        "groupingHash": "Telemetry error/nTimeout awaiting 'request' for 1ms"␊
```

Comment:
The logged timeouts should be `1200ms` but this is being replaced as part of - https://github.com/netlify/build/blob/0f6718fdaabfba02fbd7b81047e9f72eb75c3ee1/packages/build/tests/helpers/normalize.js#L105